docs/tutorial/part-zero/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Programming Basics (Part Zero)
+typora-copy-images-to: ./
+---
+
+Here’s a list of basic programming knowledge and tools that are necessary before moving on in the Gatsby tutorials. If you are new to any of these things, we've provided links to helpful tutorials on these subjects!
+
+### Necessary tools:
+* [The basics of how a website works](https://www.codeschool.com/beginners-guide-to-web-development/how-does-a-website-work) might be helpful.
+* *Command Line:* The command line is a text interface you can open on your computer. It allows you to run commands to your computer's operating system. It's necessary to learn this skill to become a programmer. [Codecademy's Command Line tutorial](https://www.codecademy.com/courses/learn-the-command-line/lessons/navigation/exercises/your-first-command) for Mac and Linux users, and [this tutorial](https://www.computerhope.com/issues/chusedos.htm) for Windows users. Even if you are a Windows user, the first page of the Codecademy tutorial is still worth reading because it explains _what_ the command line is, not just how to use it.
+* *Code editor:* Code editing programs allow you to write, edit, and save the code you'll need to write to make your website work. [Visual Studio Code](https://code.visualstudio.com/download) is a great code editor for Mac, Linux, and Windows users. Refer to their docs for help on getting started. There are many other great code editors; you may want to ask other programmers what code editor they prefer.
+* *Browser console:* Any internet browser you use has a console that allows you to inspect what is happening on any web site. This will help you figure out how to solve errors that happen while you build your website. If you are using Chrome as your web browser, here is an explanation of [how the Chrome browser console works](https://developers.google.com/web/tools/chrome-devtools/console/). If you use another web browser, search for instructions on how to access and use that browser's console.
+
+### Helpful, yet not necessary:
+* Some knowledge of HTML, CSS, and JS can be helpful. There are plenty of excellent tutorials out there; [Codecademy](https://www.codecademy.com/learn) is a good way to go.
+* *[React](https://reactjs.org/)* is a JavaScript library that Gatsby uses. Prior React experience is helpful but not necessary. In fact, many people are using Gatsby as a way to learn React.
```

Comment:
This feels like a bit of an understatement. 😅 

> React is a JavaScript library that Gatsby uses.

Maybe something more like:

> React is the JavaScript framework that Gatsby uses to build pages and structure content.

Mostly I just want this to imply a stronger relationship between Gatsby and React.